site/zh_TW.all.json

```diff
@@ -1319,5 +1319,20 @@
   },
   "shopping_card_update_reason_expired": {
     "other": "更新您的信用卡。支持的卡是 Visa、Mastercard 和 American Express。"
+  },
+  "shopping_discount_applied_promo": {
+    "other": "代碼成功應用。"
+  },
+  "shopping_error_promo_only_purchases_must_be_100_percent_discounted": {
+    "other": "請輸入有效的促銷代碼。"
+  },
+  "shopping_complete_promo_only": {
+    "other": "成功！您現在可以觀看{{.Title}} 。"
```

Comment:
Is that space supposed to be there before the dot?

Reply:
Removed. TIL this is a different character than a period, it must've escaped my find and replace